CMakeLists.txt

```diff
@@ -19,6 +19,12 @@ cable_set_build_type(DEFAULT Release CONFIGURATION_TYPES Debug;Release;Coverage)
 
 project(fizzy LANGUAGES CXX)
 
+include(TestBigEndian)
+test_big_endian(is_big_endian)
+if(is_big_endian)
+        message(FATAL_ERROR "${PROJECT_NAME} currently does not support big endian systems.")
```

Comment:
```suggestion
    message(FATAL_ERROR "${PROJECT_NAME} currently does not support big endian systems.")
```

Reply:
Can you add a cmake-format for cmakefiles? :)

Reply:
It actually exists: https://github.com/cheshirekow/cmake_format
But my editor is good enough.